e-moe/guzzle6-bundle/1.1/etc/packages/emoe_guzzle_bundle.yaml

```diff
@@ -0,0 +1,4 @@
+emoe_guzzle:
+    log:
+        enabled: true # Logging requests to Monolog
+        format: 'Guzzle: [{ts}] "{method} {uri} HTTP/{version}" {code}' # Optional log format customization
```

Comment:
yes, it have default values and these settings are optional in my bundle. should I remove them from this file?